src/com/owncloud/android/ui/activity/FileDisplayActivity.java

```diff
@@ -912,11 +918,14 @@ protected void onResume() {
         mLocalBroadcastManager.registerReceiver(mDownloadBroadcastReceiver, downloadIntentFilter);
 
         //set descending/ascending sort
-        if(descendingMenuItem != null) {
+        if(mDescendingMenuItem != null) {
```

Comment:
You are checking the `Descending` checkbox twice, at this point and [here](mDescendingMenuItem.setChecked)